src/client/pages/Forever/components/LanguagePicker.tsx

```diff
@@ -35,20 +36,21 @@ export const LanguagePicker: React.FC = () => {
   const urlParams: { locale: string; code?: string } = useParams()
   const code = urlParams.code ? urlParams.code : ''
 
-  if (market === Market.No) {
-    return (
-      <LanguagePickerWrapper>
-        <Language to={`/no/forever/${code}`}>No</Language>
-        <Divider />
-        <Language to={`/no-en/forever/${code}`}>En</Language>
-      </LanguagePickerWrapper>
-    )
-  }
   return (
     <LanguagePickerWrapper>
-      <Language to={`/se/forever/${code}`}>Sv</Language>
-      <Divider />
-      <Language to={`/se-en/forever/${code}`}>En</Language>
+      {languagePickerData[market]
+        .map((item) => (
+          <Language key={item.linkTo} to={`/${item.linkTo}/forever/${code}`}>
+            {item.shortTitle}
+          </Language>
+        ))
+        .reduce((prevOptions, option) => (
```

Comment:
Is the `reduce` just to add the divider correctly? IMO that might be unnecessarily complex. What do you think about using `index` and conditionally render the `<Divider />` based on that? Something like
```suggestion
        .map((item, index) => (
        <>
            <Language key={item.linkTo} to={`/${item.linkTo}/forever/${code}`}>
              {item.shortTitle}
            </Language>
            {index === 0 && <Divider />}
          </>
        ))
```
Or am I missing some other awesomeness with the `reduce`?

Reply:
Yeah I agree, it was a test but I think you solution is more straight forward so let's go with that!

Reply:
Oh yes of course, the `key` 🤦‍♀️